tests/example/jsonfile/jsonfile.dl

```diff
@@ -0,0 +1,16 @@
+.type List = [
+    head : number,
+    tail : List
+]
+.decl A(x : List)
```

Comment:
JSON file input seems to work, so extending the test to include input from a json file would also be good to have.